io/plugins/eu.esdihumboldt.hale.io.csv/plugin.xml

```diff
@@ -53,9 +53,9 @@
                ref="eu.esdihumboldt.hale.io.csv">
          </contentType>
          <providerParameter
-               description="States if the first line in the CSV file should be skipped (e.g. because it contains the header)"
-               label="Skip first line"
-               name="skip"
+               description="States how many lines should be skipped when reading instances from a CSV file (e.g. because of the header)"
+               label="Skip N lines"
+               name="skipN"
                optional="true">
             <parameterBinding
                   class="java.lang.Boolean">
```

Comment:
The binding should then probably be Integer?

Further down the `valueDescriptor` is also not updated yet.